src/components/shared/NewEventForm.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react';
```

Comment:
Here you can also import `useState` directly : `import React, { useState } from 'react';`